crates/common/logged_command/src/logged_command.rs

```diff
@@ -260,15 +272,15 @@ mod tests {
 
         // Prepare a command
         let mut command = LoggedCommand::new("echo");
-        command.arg("Hello").arg("World!");
+        command.args(["Hello", "World!"]);
 
         // Execute the command with logging
         let _ = command.execute(&mut logger).await;
 
         let log_content = String::from_utf8(std::fs::read(log_file_path)?)?;
         assert_eq!(
             log_content,
-            r#"----- $ echo "Hello" "World!"
+            r#"----- $ echo ["Hello", "World!"]
```

Comment:
The assert is unexpected, as it is printed invalid shell output (making it harder for users to debug by copying/pasting the output)...

```
$ echo Hello World
```

Alternatively you could not try to replicate a shell output and just use the following format (which does not to mix shell with Rust formatted output).

```
cmd=echo, args=["Hello", "World!"]
```

Though the above example is not so user friendly if people want to copy/paste the command to help with debugging (by manually running the script and checking the output).

Reply:
It might be worthwhile checking out the workflow actor does the logging of the commands and using the same convention.